### DISCUSSION TITLE
`expo-constants` with the Next app

### DISCUSSION THREAD

**Discussion by bombillazo** [nandorojo/solito#251] [Q&A]
Hey there, how do we pass the app.config.json used by Expo to the Next.js app? We use `expo-constants` to get some configuration variables but there is not equivalent config on Next.js.

**Comment by chrisarts** [comment]
In our project we use a mix of them, 
- create an env config file with those values centralized, 
- here you have two options .web.ts or .native.ts or create helper functions.
- use the same .env you use on nextJS and inject those values in expo constants to have a single source of truth using the app.config.js file.

**Reply by chrisarts** [reply]
Also for the .env file I create symliks from the root folder to allow prisma detected too and the same symlink in expo
